cvat/apps/annotation/serializers.py

```diff
@@ -29,6 +37,34 @@ def create(self, validated_data):
 
         return annotation_format
 
+    # pylint: disable=no-self-use
+    def update(self, instance, validated_data):
+        handler_names = [handler["display_name"] for handler in validated_data["handlers"]]
+
+        handlers_to_delete = [ db_handler for db_handler in instance.annotationhandler_set.all() if db_handler.display_name not in handler_names]
+        for db_handler in handlers_to_delete:
+            db_handler.delete()
+
+        handlers_to_create = []
+        for handler in validated_data["handlers"]:
+            updated = False
+            for db_handler in instance.annotationhandler_set.all():
+                if handler["display_name"] == db_handler.display_name:
+                    serializer = AnnotationHandlerSerializer(db_handler, data=handler)
+                    if serializer.is_valid(raise_exception=True):
+                        serializer.save()
+                        updated = True
+                        break
+            if not updated:
+                handlers_to_create.append(handler)
+
+        handlers_to_create = [models.AnnotationHandler(annotation_format=instance, **handler) for handler in handlers_to_create]
+        if handlers_to_create:
+            models.AnnotationHandler.objects.bulk_create(handlers_to_create)
```

Comment:
Do you really want to have bulk_create here? Is it better to simplify logic and use the same code to create and update?